model/slack_attachment.go

```diff
@@ -51,7 +49,7 @@ func ExpandAnnouncement(text string) string {
 // can be found in the text attribute, or in the pretext, text, title and value
 // attributes of the attachment structure. The Slack attachment structure is
 // documented here: https://api.slack.com/docs/attachments
-func (a *SlackAttachments) Process() {
+func ProcessSlackAttachments(a *[]*SlackAttachment) {
```

Comment:
Pick one:

`o.Attachments = ProcessSlackAttachments(o.Attachments)`

`ProcessSlackAttachments(&o.Attachments)`

I chose the latter. My opinions on the matter are very weak though.